tests/rapid_gossip_sync_test.rs

```diff
@@ -0,0 +1,101 @@
+mod setup;
+
+// Caution: Run these tests sequentially, otherwise they will corrupt each other
+//      because they are manipulating their environment:
+// cargo test --features nigiri -- --test-threads 1
```

Comment:
```suggestion
// Caution: Run these tests sequentially, otherwise they will corrupt each other
//          because they are manipulating their environment:
//          cargo test --features nigiri -- --test-threads 1
```